src/prefect/engine.py

```diff
@@ -191,14 +197,22 @@ async def create_then_begin_flow_run(
         raise RuntimeError(
             f"Cannot create flow run. Failed to reach API at {client.api_url}."
         ) from connect_error
-
     state = Pending()
     if flow.should_validate_parameters:
         try:
             parameters = flow.validate_parameters(parameters)
+        except (ParameterTypeError, SignatureMismatchError) as exc:
+            validation_failure_msg = str(exc)
```

Comment:
minor nit -- I always prefer `repr(exc)` for exceptions, as the repr preserves the exception type whereas the string representation is _only_ the message.

Also I think you meant to pass this variable in as the `message` input to the state below 👍